pilot/pkg/model/gateway_test.go

```diff
@@ -1,197 +0,0 @@
-// Copyright 2018 Istio Authors.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package model
```

Comment:
It'd be nice to keep at least some unit tests to verify some of the invariants in the merging...we're going to end up with bugs due to this logic down the line.

Reply:
Oh I removed this because this file was completely skipped.. Look at the test.Skip in this file. :).
All data structs and assumptions in this file seemed to be pointing to the very old version of gateway merging that existed before v2